store/userStore.ts

```diff
@@ -0,0 +1,9 @@
+import { create } from 'zustand';
+
+const userStore = ( ) => ({
+    example: []
+});
+
+const useUser = create(userStore);
```

Comment:
This is incorrectly named. This should be `useUserStore`, or what ever we renamed the store to.